.github/workflows/buildamd64.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
why separate the build scripts? using buildx for both simplifies the process. also, the eventual plan is to start using YAML anchors to shorten docker.yml (once GitHub actions support them) so we should really try to keep both jobs the same.